docs/office/leave.rst

```diff
@@ -74,3 +74,25 @@ Unused leaves
   up to a total of 5 days.
   Try to plan your time so that you don't end up
   with these remaining days in the first place.
+
+
+Meet in person
+==============
+
+In person interaction is important for any team.
+As all team members are working from different places and do not
+get the chance to have the coffee chit chat or the idle work breaks, 
+we try to get the team together at least once a year.
+Usually it's for a week, each time in a different location.
+
+During these periods work sessions are alternated with get to know each other
```

Comment:
I'm thinking:

`During the team building meeting, work sessions are alternated with hangouts to get to know each other.`